keyboards/fractal/keymaps/default/config.h

```diff
@@ -0,0 +1,42 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the first three lines (up to and including the include) with `#pragma once` instead, and remove the `#endif` at the end?